nautobot/extras/models/customfields.py

```diff
@@ -315,18 +312,15 @@ class CustomField(BaseModel, ChangeLoggedModel, NotesMixin):
         default=CustomFieldTypeChoices.TYPE_TEXT,
         help_text="The type of value(s) allowed for this field.",
     )
-    # 2.0 TODO: #824 remove `name` field as redundant, make `label` mandatory, populate `slug` from `label` field.
-    name = models.CharField(max_length=50, unique=True, help_text="Human-readable unique name of this field.")
     label = models.CharField(
         max_length=50,
-        blank=True,
-        help_text="Name of the field as displayed to users (if not provided, the field's name will be used.)",
+        help_text="Name of the field as displayed to users.",
     )
-    slug = AutoSlugField(
+    key = AutoSlugField(
```

Comment:
Wondering out loud: should we also rename `ComputedField.slug` and `Relationship.slug` for the same reasoning and self-consistency?

Reply:
We were talking about it yesterday, we def should. Gonna make an issue after standup.